### PR TITLE
RELATED: RAIL-3963 Proofread the changes from PR 494

### PR DESCRIPTION
**Pull request by evgeniabogranskaya**
https://github.com/gooddata/gooddata-ui/pull/494/

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1000/ : SUCCESS in 2m 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/667/ : SUCCESS in 9s

File: docs/18_dashboard_plugins_api.md [hunk 1]
```diff
@@ -166,23 +166,21 @@ to add extra elements on top of the widgets.
 The KPI customization API contains the `withCustomProvider()` and `withCustomDecorator()` methods that behave the same way
 the [insight widget methods](18_dashboard_plugins_api.md#customize-rendering-of-insights) do.
 
-### Custom widgets
+### Customize widgets
```
Comment (no23reason) [inline docs/18_dashboard_plugins_api.md:169]: These really are Custom widgets, you can customize widgets even using other parts of the API.

Comment (evgeniabogranskaya) [reply docs/18_dashboard_plugins_api.md]: fixed to say "Use custom widgets"

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1001/ : SUCCESS in 1m 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/668/ : SUCCESS in 10s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR500

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/1002/ : SUCCESS in 1m 23s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/203/ : SUCCESS in 55s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/98/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/293/ : SUCCESS in 21s

**Pull request merged**